app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageComposerViewModel.kt

```diff
@@ -33,17 +33,7 @@ import com.wire.android.mapper.ContactMapper
 import com.wire.android.media.PingRinger
 import com.wire.android.model.ImageAsset.PrivateAsset
 import com.wire.android.model.SnackBarMessage
-import com.wire.android.navigation.EXTRA_CONVERSATION_ID
-import com.wire.android.navigation.EXTRA_GROUP_DELETED_NAME
-import com.wire.android.navigation.EXTRA_LEFT_GROUP
-import com.wire.android.navigation.EXTRA_MESSAGE_TO_DELETE_ID
-import com.wire.android.navigation.EXTRA_MESSAGE_TO_DELETE_IS_SELF
-import com.wire.android.navigation.NavigationCommand
-import com.wire.android.navigation.NavigationItem
-import com.wire.android.navigation.NavigationManager
-import com.wire.android.navigation.SavedStateViewModel
-import com.wire.android.navigation.getBackNavArg
-import com.wire.android.navigation.getBackNavArgs
+import com.wire.android.navigation.*
```

Comment:
**question**: Should we be adding `.*` in imports? instead of the full path and etc

Reply:
already removed, thanks.